app/src/main/java/app/grapheneos/pdfviewer/PdfViewer.java

```diff
@@ -393,6 +390,22 @@ public void onZoomEnd() {
         }
     }
 
+    @Override
+    protected void onNewIntent(Intent intent) {
```

Comment:
`onNewIntent` won't ever be called when `android:documentLaunchMode="always"` is set.
A new instance of `PdfViewer` activity will be created each time.

Reply:
okay,